index.js

```diff
@@ -22,7 +22,11 @@ exports.init = function(opts) {
   exports.SchemaAdapter = require('./adapters/mongoose');
 
   exports.getModels = function () {
-    return opts.mongoose.models;
+    if ((typeof opts.getModels) === 'function') {
```

Comment:
@arnaudbesnier : up :smile:  (I didn't tag you in my previous question so maybe you didn't see it)